radixdlt/src/main/java/com/radixdlt/middleware2/network/GetVerticesRequestMessage.java

```diff
@@ -28,27 +28,37 @@
 /**
  * RPC Message to get request for a vertex
  */
-@SerializerId2("message.consensus.vertex_request")
-public final class GetVertexRequestMessage extends Message {
+@SerializerId2("message.consensus.vertices_request")
+public final class GetVerticesRequestMessage extends Message {
 	@JsonProperty("vertexId")
 	@DsonOutput(Output.ALL)
 	private final Hash vertexId;
 
-	GetVertexRequestMessage() {
+	@JsonProperty("count")
+	@DsonOutput(Output.ALL)
+	private final int count;
```

Comment:
Is this a required count or more of a limit?

Reply:
required